kubernetes/cluster_loadtest.go

```diff
@@ -129,9 +130,15 @@ func (c *Cluster) Loadtest(options *ltops.LoadTestOptions) error {
 		return errors.Wrap(err, "unable to get app pods")
 	}
 
-	err = c.bulkLoad(loadtestPods[0], appPods[0], options.ForceBulkLoad)
-	if err != nil {
-		return err
+	if options.SkipBulkLoad {
+		if !c.Configuration().BulkLoadComplete {
+			log.Info("Bulk loading not complete, you may need load that, if you loaded outside the ltops tool you can proceed otherwise might have wrong results")
```

Comment:
Maybe change this to `Bulk loading skipped. If you have not previously loaded the data the test may result in errors.`

Reply:
done. thanks!